### DISCUSSION TITLE
1.1.20

### DISCUSSION THREAD

**Discussion by absidue** [wstyres/Zebra#1877] [Releases]
This release significantly reduces the size of Zebra, this means you will be able to download Zebra quicker and it will take up less space on your device.

No bugs were fixed and no features were added or removed.

Technical changes that lead to the size reduction:
* Fully remove Crashlytics (it was already disabled in stable releases)
* Disable the embedding of bitcode
* Shrinking the image assets

Thanks to @elihwyma for the bitcode advice and image shrinking.

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.20'>1.1.20</a>.</em>

**Comment by adminy** [comment]
![5340D933-001B-4BFF-908D-1B1F7EB84F2B](https://user-images.githubusercontent.com/22717869/128258750-b4a6c378-9d13-41d1-aa8a-9d486b1c8d26.png)

Works with 14.7.1
